src/index.ts

```diff
@@ -66,6 +67,8 @@ type IReturnData<O extends Options = {}> = (IDomainData) & (IReturnCa<O>) & (IRe
  * as { caPath: string }
  */
 export async function certificateFor<O extends Options>(domain: string, options: O = {} as O): Promise<IReturnData<O>> {
+  if (VALID_IP.test(domain)) {
+    throw new Error('IP addresses are not supported currently');
```

Comment:
I think a `}` was missed here
```suggestion
  if (VALID_IP.test(domain)) {
    throw new Error('IP addresses are not supported currently');
  }
```